### PR TITLE
added  two new commands for  exporting and importing .xml files

### PR DESCRIPTION
**Pull request by RAJU2529**
Taken example from Windows 10 v1903  build no 18362.207. 
added  two commands  under export of configuration file  and  import of configuration  file

add this exploit configuration file.xml is  created by me  with windows 10 v1903
 
[ExploitConfigfile.zip](https://github.com/MicrosoftDocs/windows-itpro-docs/files/3338834/ExploitConfigfile.zip)

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-exploit-guard/import-export-exploit-protection-emet-xml.md [hunk 1]
```diff
@@ -70,6 +70,9 @@ When you have configured exploit protection to your desired state (including bot
 
 Change `filename` to any name or location of your choosing.
 
+example command
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-exploit-guard/import-export-exploit-protection-emet-xml.md]:
```suggestion
Example command
```

File: windows/security/threat-protection/windows-defender-exploit-guard/import-export-exploit-protection-emet-xml.md [hunk 2]
```diff
@@ -91,6 +94,10 @@ After importing, the settings will be instantly applied and can be reviewed in t
 
 Change `filename` to the location and name of the exploit protection XML file.
 
+example command
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-exploit-guard/import-export-exploit-protection-emet-xml.md]:
```suggestion
Example command
```

**Comment by e0i** [comment]
@levinec A community contributed PR that has passed copy review is available for technical review. Thanks.

**Pull request merged**